togetherly/lib/models/chore.dart

```diff
@@ -0,0 +1,26 @@
+// ignore_for_file: constant_identifier_names
+
+enum ChoreStatus {
+  ASSIGNED,
```

Comment:
The Flutter convention is [lowerCamelCase for constants](https://dart.dev/effective-dart/style#prefer-using-lowercamelcase-for-constant-names). If we want to adopt a different convention, I'd rather change our linter rules than disable the linter rules on specific files.

Reply:
I'm fine changing the constants to match the convention. Does that convention not include regular variables and function names too though? lowerCamelCase seems pretty standard for most variables, I always thought that constants were intentionally different from that.

Reply:
The link in my previous comment explains their reasoning. I could see it either way, but I do think we should configure the linter to match whatever convention we choose to follow.